lib/vanagon/platform.rb

```diff
@@ -8,7 +8,7 @@ class Platform
     attr_accessor :docker_image, :ssh_port, :rpmbuild, :install, :platform_triple
     attr_accessor :target_user, :package_type, :find, :sort, :build_hosts, :copy, :cross_compiled
     attr_accessor :aws_ami, :aws_user_data, :aws_shutdown_behavior, :aws_key_name, :aws_region, :aws_key
-    attr_accessor :aws_instance_type, :aws_vpc_id, :aws_subnet_id
+    attr_accessor :aws_instance_type, :aws_vpc_id, :aws_subnet_id, :output_dir
```

Comment:
I'd probably set `:output_dir` as an `attr_writer` because the corresponding accessor method is being explicitly defined, but that's the pedantry talking.